htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -101,31 +104,6 @@ class FactureFournisseur extends CommonInvoice
     var $extraparams=array();
 
     /**
-     * Standard invoice
-     */
-    const TYPE_STANDARD = 0;
-
-    /**
-     * Replacement invoice
-     */
-    const TYPE_REPLACEMENT = 1;
-
-    /**
-     * Credit note invoice
-     */
-    const TYPE_CREDIT_NOTE = 2;
-
-    /**
-     * Deposit invoice
-     */
-    const TYPE_DEPOSIT = 3;
-
-    /**
-     * Proforma invoice
-     */
-    const TYPE_PROFORMA = 4;
```

Comment:
They are in CommonInvoice and they were replacing the parent object values.
So they are still available :)

El lun., 16 de marzo de 2015 18:00, Raphaël Doursenaud <
notifications@github.com> escribió:

In htdocs/fourn/class/fournisseur.facture.class.php
> <https://github.com/Dolibarr/dolibarr/pull/2456#discussion_r26504559>:
>
> > -    const TYPE_REPLACEMENT = 1;
> > -
> > -    /**
> > -     * Credit note invoice
> > -     */
> > -    const TYPE_CREDIT_NOTE = 2;
> > -
> > -    /**
> > -     * Deposit invoice
> > -     */
> > -    const TYPE_DEPOSIT = 3;
> > -
> > -    /**
> > -     * Proforma invoice
> > -     */
> > -    const TYPE_PROFORMA = 4;
>
> It's a shame to lose those constants. I found them really useful since
> they made code much more readable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Dolibarr/dolibarr/pull/2456/files#r26504559>.
>